phlay

```diff
@@ -61,8 +61,16 @@ class UserError(Exception):
 class Conduit:
     def __init__(self, arcrc_path):
         # Load the global and local Arcanist configuration
-        self.top = Path(check_output(['git', 'rev-parse', '--show-toplevel'])
-                        .decode().strip())
+        git_top = (check_output(['git', 'rev-parse', '--show-toplevel'])
+                   .decode().strip())
+        if (len(git_top) > 2 and git_top[0].isalpha() and git_top[1] == ":" and
```

Comment:
Hmm. `git rev-parse` always returns an absolute path IIRC, so we could probably simplify this to:

```python
if git_top.startswith('/'):
    self.top = Path(git_top)
else:
    # If we're in WSL this may be due to it being a Windows path, try converting it.
    try:
        self.top = Path(check_output(['/bin/wslpath', git_top]).decode().strip())
    except FileNotFoundError:
        raise UserError("unable to locate git root")
```

Reply:
git rev-parse does always return an absolute path. However, unless I'm missing something, your proposed code would fail if someone was running both git and phlay in Windows natively (no WSL or mingw). In that case, it would hit the else clause, then fail when `wslpath` was not found.

Reply:
Hmm, true. Perhaps we could just fall back to `git_top` in the `FileNotFoundError` case instead of raising a `UserError`?